docs/_posts/murat-gunay/2021-12-06-roberta_token_classifier_icelandic_ner_is.md

```diff
@@ -19,7 +19,7 @@ use_language_switcher: "Python-Scala-Java"
 
 - This model is imported from `Hugging Face`.
```

Comment:
Let's join the two sentences:
`This model, imported from Hugging Face, was fine-tuned on the MIM-GOLD-NER dataset for the Icelandic language, leveraging Roberta embeddings and using RobertaForTokenClassification for NER purposes.`